Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.py

```diff
@@ -37,21 +37,23 @@ def build_urls_dict(regions_list: list, services_list: list, unique_id) -> List[
     return urls_list
 
 
-class Client(BaseClient):
+class Client:
     """
     Client to use in the Office 365 Feed integration. Overrides BaseClient.
     Office 365 IP address and URL web service announcement:
     https://docs.microsoft.com/en-us/office365/enterprise/managing-office-365-endpoints?redirectSourcePath=%252fen-us%252farticle%252fmanaging-office-365-endpoints-99cab9d4-ef59-4207-9f2b-3728eb46bf9a#webservice
     https://techcommunity.microsoft.com/t5/Office-365-Blog/Announcing-Office-365-endpoint-categories-and-Office-365-IP/ba-p/177638
     """
 
-    def __init__(self, urls_list: list, insecure: bool = False):
+    def __init__(self, urls_list: list, insecure: bool = False, tags: Optional[list] = None):
```

Comment:
lets stay consistent with `Dict` and `List` instead of `dict` and `list`

Reply:
But they are not the same things.
dict means it returns a dict while Dict requires to fill the structure of the dict itself.
`var: dict`
vs
`var: Dict[str, str]`

And it's just not clear to do things like:
`var: List[Any]`, it's confusing